github/repos_comments.go

```diff
@@ -109,6 +116,9 @@ func (s *RepositoriesService) GetComment(owner, repo string, id int) (*Repositor
 		return nil, nil, err
 	}
 
+	// TODO: remove custom Accept header when this API fully launches.
+	req.Header.Set("Accept", mediaTypeReactionsPreview)
+
 	c := new(RepositoryComment)
 	resp, err := s.client.Do(req, c)
 	if err != nil {
```

Comment:
> I've added the `ListOptions` and updated the tests, but I don't think any docs need updating based upon the docs found in your example from `activity_events.go.`

Sounds good.

> According to the docs for `UpdateComment`:
https://developer.github.com/v3/repos/comments/#update-a-commit-comment
it does not return the reactions object.

Oh, I see now that the following 3 endpoints are actually documented with a new "Reactions summary" section to have reactions present with that accept header set:

- https://developer.github.com/v3/repos/comments/#list-commit-comments-for-a-repository
- https://developer.github.com/v3/repos/comments/#list-comments-for-a-single-commit
- https://developer.github.com/v3/repos/comments/#get-a-single-commit-comment

The `UpdateComment` endpoint indeed does not mention it.

I didn't see that before, which is why I asked that question.

> So I'm thinking it would cause less user confusion if we leave off the custom Accept header and let the user get an error if they provide a Reactions struct in the provided RepositoryComment instead of accepting it and giving them the false hope that they can update it.

> Thoughts?

Full agreement. You should follow the documentation that only mentions the 3 endpoints, and not add it for `UpdateComment`.